.tools-version

```diff
@@ -0,0 +1 @@
+golang 1.18.4
```

Comment:
what actually is this file for? is it for an intellij IDEs?

Reply:
This is for `asdf`, to have multiple versions of golang installed (or a lot of other tools).
https://asdf-vm.com/